### PR TITLE
change order of informations presented by CTRL+NVDA+f1, to make changeable info presented first.

### LINKED ISSUE #7338: ctrl+NVDA+f1 Speech Order
Hitting ctrl+NVDA+f1, we get something like
Currently running application is firefox.exe and currently loaded module is firefox

Could we instead consider:
firefox.exe is currently running. firefox module is loaded.

This gets us the information that we want right away. I have no UI training/expertise but it seems to make sense that the part of a message that varies should be spoken first. e.g. when we select text, we hear "hello selected", rather than "selected hello", because we know that we're selecting text, so the part that varies comes first and is the actual text selected.

### PR DESCRIPTION
**Pull request by lukaszgo1**
### Link to issue number:
Fixes #7338
### Summary of the issue:
When pressing CTRL+NVDA+F1 the order of information's is not optimal.
### Description of how this pull request fixes the issue:
The changeable parts i.e. application name, and app module name are now presented first.
### Testing performed:
Tested, that script still present the same info, but in the different order.
### Known issues with pull request:
None
### Change log entry:
This command isn't even included in the user guide, so i believe, that Change log entry isn't nesesary.

### CODE REVIEW

**Comment by jscholes** [comment]
I think this change itself is minor, and as you mentioned the command isn't mentioned in the user guide.  However, there is no seperator between your two messages, be it a comma, a similar character or the word "and".  This results in strings like:

> explorer.exe is currently running Explorer module is loaded

with the information running together.

**Comment by lukaszgo1** [comment]
@jscholes Thanks for your commends. I assumed, that the space at the beginning of the second message would be sufficient, but adding a dot after the first looks like a good solution.

**Comment by leonardder** [comment]
Now while we're at it, I think I prefer swapping the order as well. First report the app module, than the executable name. This makes it more intuitive to hear whether a specific appmodule is loaded for an app.

**Comment by lukaszgo1** [comment]
It'd be nice to have this reviewed.

**Comment by feerrenrut** [comment]
> It'd be nice to have this reviewed.

I'm looking at it now. In the future, please use the "request review" button. Though saying that, do you have access to it?

**Comment by lukaszgo1** [comment]
No i don't as per this commend in the GitHub documentation 

> Note: Pull request authors can't request reviews unless they are either a repository owner or collaborator with write access to the repository.

**Comment by feerrenrut** [comment]
Ah, I see. My apologies.

File: source/globalCommands.py [hunk 1]
```diff
@@ -1569,17 +1569,17 @@ def script_passNextKeyThrough(self,gesture):
...
-			# Translators: Indicates the name of the appModule for the current program (example output: and currently loaded module is explorer).
-			# For example, the complete message for Windows explorer is: Currently running application is explorer.exe and currently loaded module is explorer.
+			# Translators: Indicates the name of the appModule for the current program (example output: explorer module is loaded).
 			# This message will not be presented if there is no module for the current program.
-			message += _(" and currently loaded module is %s") % mod.appModuleName.split(".")[0]
+			message = _(" %s module is loaded. ") % mod.appModuleName.split(".")[0]
+		appName=appModuleHandler.getAppNameFromProcessID(focus.processID,True)
+		# Translators: Indicates the name of the current program (example output: explorer.exe is currently running).
+		# Note that it does not give friendly name such as Windows Explorer; it presents the file name of the current application.
+			# For example, the complete message for Windows explorer is: Explorer.exe is currenty running. explorer module is loaded..
```
Comment (feerrenrut) [inline source/globalCommands.py]: There is an extra tab here, and the order of the messages in the example is not correct. Please also put quote characters around the example to make it clear.

```suggestion
		# EG: "explorer module is loaded. Explorer.exe is currenty running."
```

**Review by feerrenrut — CHANGES_REQUESTED** [review]

**Comment by leonardder** [comment]
[**@feerrenrut**](https://github.com/feerrenrut) commented on [9 May 2019, 19:22 CEST](https://github.com/nvaccess/nvda/pull/8927#issuecomment-490992334 "2019-05-09T17:22:38Z - Replied by Github Reply Comments"):
> I'm looking at it now. In the future, please use the "request review" button. Though saying that, do you have access to it?

Unfortunately, only collaborators can access this.

**Comment by lukaszgo1** [comment]
@feerrenrut I believe I have addressed your commend.

**Review by feerrenrut — APPROVED** [review]
Thanks @lukaszgo1

**Comment by feerrenrut** [comment]
There should still be a change log entry. I will add something along the lines of:

**changes for developers**
```
When using the report module info command, the order of information has changed to present the module first. (#7338)
```

**Pull request merged**